.eslintrc.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  extends: 'airbnb',
+  globals: {
+    Atomics: 'readonly',
+    SharedArrayBuffer: 'readonly',
```

Comment:
@aku2nnya @saeed79darvish I suspect that several configurations in this config file are extraneous. I don't think that the globals "Atomics" and "SharedArrayBuffer" will be used during FEC. Also, I could be wrong, but I think that everything in "parserOptions" and "plugins" is unnecessary. "rules" appears to be empty, was that intentional?

Reply:
@jordanstholmes 
Yes, it appears that since we are using the 'react' plugin, the above are not necessary (if I'm understanding the [docs](https://eslint.org/docs/user-guide/configuring) correctly at least).
Will remove these in a future branch/PR.
For 'rules' I believe we are suppose to leave this blank since we want to use the airbnb style and not override it with our custom rules, but am I understanding that incorrectly?

Reply:
@aku2nnya I think the eslint-config npm package handles many of those settings. The rules section can be useful for turning off certain rules off that you don't intend to adhere to (so you'll probably find it useful at some point), but I don't believe it does anything when it's empty.